doc/api/dns.md

```diff
@@ -159,6 +162,8 @@ Create a new resolver.
     each name server before giving up. **Default:** `4`
   * `maxTimeout` {integer} The max retry timeout, in milliseconds.
     **Default:** `0`, disabled.
+  * `rotate` {boolean} Perform round-robin selection of the nameservers for each resolution.
```

Comment:
Why don't we go with `selectionStrategy: 'round-robin' | undefined` rather than a boolean flag?